micro-rdk/src/common/grpc.rs

```diff
@@ -1246,6 +1251,11 @@ where
         Err(ServerError::from(GrpcError::RpcUnavailable))
     }
 
+    #[cfg(feature = "camera")]
+    fn camera_get_frames(&mut self, _message: &[u8]) -> Result<(), ServerError> {
```

Comment:
if theses functions (including point cloud, render frame etc.. are going to remain unimplemented we should remove the code because by default we return unimplemented on all unknown RPCs

Reply:
So remove unimplemented rpcs from `grpc.rs` but leave the trait methods in `camera.rs` (returning an error/leaving them blank)?